src/Activity/TrainWithExample/index.js

```diff
@@ -24,7 +25,8 @@ class TrainWithExample extends React.Component {
     this.newActivityRef.child("activity_type").set("example");
     const image = {
       mammals: mammalsData[this.state.index].src,
-      parallelograms: parallelogramData[this.state.index].src 
+      parallelograms: parallelogramData[this.state.index].src,
+      adverbs: adverbsData[this.state.index].adverb 
```

Comment:
This part is crashing. If `this.state.index` is larger than the length of  `mammalsData` it creates the error: `Uncaught TypeError: Cannot read property 'src' of undefined`.